[Studios] Session duration (#486)
seqeralabs/docs: platform_versioned_docs/version-24.3/data_studios/managing.mdx

        - **Data studio name**
        - Optional: **Description**
     1. Optional: Select **Install Conda packages** to enter or upload a list of Conda packages to include with the data studio. For more information on the syntax for specifying Conda packages, see [Conda package syntax][conda-syntax].
        - **Session lifespan**: Depending on your workspace settings, you may be able to choose between the following options.
          -  **Stop the session automatically after a predefined period of time.**

>                - If there is a existing defined session lifespan workspace setting, you won't be able to edit this. If no workspace setting is defined, you can edit this field. The default value is 8 hours.

Author: ```suggestion
               - If there is an existing defined session lifespan workspace setting, you won't be able to edit this. If no workspace setting is defined, you can edit this field. The minimum value is 1 hour and the maximum is 120 hours. The default value is 8 hours. 
```

>                - If you increase this default value, it applies only to that session. Once you've stopped the session, the value returns to default.

Reviewer: minor: `If you change this default value, ...` because user can also decrease, to have a shorter then 8 hour session

>                - If you increase this default value, it applies only to that session. Once you've stopped the session, the value returns to default.

Author: ```suggestion
               - If you change the default value, the change applies only to that session. Once you've stopped the session, the value returns to default.
```
